codes/extractive/netsum/netsum.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Oct 26 21:41:03 2018
+
+@author: Rose
+"""
+from __future__ import division
+import nltk
+from nltk.tokenize import sent_tokenize, word_tokenize, RegexpTokenizer
+import pandas as pd
+import numpy as np
+from collections import Counter
+import h5py
+import os
+import keras
+from sklearn.model_selection import train_test_split
+from keras.models import Sequential, Model
+from keras.layers import Dense, Dropout, Activation, Flatten, Conv2D, MaxPooling2D, Input
+from keras.callbacks import ModelCheckpoint, EarlyStopping
+
+text_path = "/root/nlp_project/cnn/text_2000/"
```

Comment:
Seems we already specify the data folder path. Could you use relative path here thus all of us can run this code?